src/main/java/net/sf/jabref/model/event/EntryChangedEvent.java

```diff
@@ -1,6 +1,5 @@
 package net.sf.jabref.model.event;
```

Comment:
Same for me. There are good reasons for both ways of structuring. @tobiasdiez Please provide a compelling argument, and we will go with your suggestion ;-)

From what I see: Sorting the events into different packages increases cohesion, but also "pollutes" the packages with more classes. What about further structuring, e.g. `model.entry.event`, `model.database.event`, `model.groups.event`?